snap/local/hooks/go.mod

```diff
@@ -1,5 +1,6 @@
 module github.com/edgexfoundry/device-rest-go/hooks
 
-require github.com/canonical/edgex-snap-hooks/v2 v2.0.3
+require github.com/canonical/edgex-snap-hooks/v2 v2.1.1
```

Comment:
Please tidy and check-in the `go.sum` as well.

Reply:
Added. Thanks!